packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -35,12 +37,15 @@ const getMarketsParamsSpecific = t.intersection([
     creator: t.string,
     category: t.string,
     search: t.string,
-    reportingState: t.union([t.string, t.array(t.string)]),
+    reportingStates: t.array(t.string),
```

Comment:
shouldn't this be typed so that if a unknown reportingState is passed it it'll get caught and return an error?

Reply:
Yeah, I was also going to do that with the sortBy param. This is meant to allow multiple reporting states, but I'll add validation for that.